content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,173 @@
+
+For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
+
+Use these instructions to build a `rippled` binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. These instructions were tested on Ubuntu 16.04 LTS.
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+Here are the **minimum** system requirements for building and running `rippled`.
+
+**_To build rippled:_**
+
+* RAM: 8GB
+
+**_To run rippled:_**
+
+* RAM: 4GB
+* CPU: 64-bit x86_64, 2+ cores
+* Disk: 50GB SSD with 500 IOPS for the database partition
+* Network: A fast network connection is preferable
+
+If you are running `rippled` in an Amazon EC2 instance, Ripple recommends selecting a **Type** of **_m3.large_**. If you choose to run a validating `rippled` server, you may need more resources.
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+```
+sudo apt-get update
+```
+2. Upgrade currently installed packages.
+```
+sudo apt-get -y upgrade
+```
+3. Install Git.
+```
+sudo apt-get -y install git
+```
+4. Install SCons.
+```
+sudo apt-get -y install scons
+```
+5. Install Ctags.
+```
+sudo apt-get -y install ctags
+```
+6. Install `pkg-config`.
+```
+sudo apt-get -y install pkg-config
+```
+7. Install Protocol Buffers.
+```
+sudo apt-get -y install protobuf-compiler
+sudo apt-get -y install libprotobuf-dev
+```
+8. Install Secure Socket Layer (SSL) toolkit development files.
+```
+sudo apt-get -y install libssl-dev
+```
+9. Install `python-software-properties`.
+```
+sudo apt-get install -y python-software-properties
+```
+10. Install Boost.
+```
+sudo add-apt-repository -y ppa:boost-latest/ppa
+sudo apt-get -y update
+sudo apt-get install -y libboost-all-dev
+```
+11. Get `rippled` source code.
+```
+git clone https://github.com/ripple/rippled.git
+cd rippled
+git checkout master
+```
+12. Build `rippled` binary executable from source code. This may take about 30 minutes.
+
+    SCons saves the built executable in `rippled/build`.
+
+```
+scons
+```
+
+
+## 2. Configure `rippled`
+
+1. Configure `rippled.cfg`.
+
+    `rippled.cfg` is the configuration file for `rippled`.
+
+    You can find the `rippled-example.cfg` example file in `rippled/doc`. See the file for a description of all configuration options. You should be able to connect to the XRP Ledger using the default configuration defined in the example file.
+
+    Put your configured `rippled.cfg` file in `etc/opt/ripple/`.
+
+2. Configure `validators.txt`.
```

Comment:
Per [DEC-745](https://ripplelabs.atlassian.net/browse/DEC-745), the default config file does not list the validators explicitly (I was basing that on a mistaken previous assumption) but instead uses `[validators_file]` to specify a file with the default validator config, with the idea being that `validators.txt` should be automatically replaced when there are updates while `rippled.cfg` should not. That's because `rippled.cfg` may contain settings that are specific and/or sensitive (like a validating server's secret or token) whereas `validators.txt` is general and everyone should generally use the same one so there's no chance of forking away from the majority of the network.

People (like me) who have a config file derived from an older (pre-0.32.0) example may have their validators explicitly configured in `rippled.cfg` instead. They should be sure to update their config when the 0.81.0 release happens. That's outside the scope of this task.

What's in-scope for this task is that the install should include copying the default `validators.txt` file to an appropriate location. If you put it in the same folder as `rippled.cfg` (which you should) then you don't even need to edit the `[validators_file]` stanza of `rippled.cfg`. So under "configure `rippled`" you'd add another step like this:

> 3. Copy the example `validators.txt` file to the same folder as `rippled.cfg`:
> 
>         cp doc/validators-example.txt ~/.config/ripple/validators.txt

Reply:
Also, unlike the language in the 1st draft which mentions "a description of all validator options", the final version should have a strongly-worded warning that you _SHOULD NOT_ modify the example validators.txt file except as directed by Ripple, or else you may fork away from the production network and could lose money.

Reply:
👍  Done - I used the language from Nik in cpp-informal and included a link to the decentralization strategy update provided by Arthur.